From 36a8b586946dc64932431900502fcca597f01ec2 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Fri, 20 Oct 2017 23:49:42 +0800
Subject: [PATCH 2670/5242] MLK-16676-2 ARM64: dts: freescale: imx8qm: add
 early_power_on for intmux

commit  25222dcd7587d90a280b329e5ca7f67a6281b8e7 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX8QM, intmux is registered as irq chip driver,
it resumes earlier then generic power domain, so need
to add early_power_on property.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 1bda96a..88576f7 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -989,6 +989,7 @@
 				reg = <SC_R_M4_0_INTMUX>;
 				#power-domain-cells = <0>;
 				power-domains =<&pd_cm40>;
+				early_power_on;
 			};
 		};
 
@@ -1015,6 +1016,7 @@
 				reg = <SC_R_M4_1_INTMUX>;
 				#power-domain-cells = <0>;
 				power-domains =<&pd_cm41>;
+				early_power_on;
 			};
 		};
 	};
-- 
1.7.9.5

